### DISCUSSION TITLE
Packaging Directory Issues

### DISCUSSION THREAD

**Discussion by 1893945** [pyinstaller/pyinstaller#7423] [Help]
I want to generate all the dependent files (dll, pyd) to a specified directory. Only the exe files are placed in the packaging directory. What should I do?

**Comment by rokm** [comment]
Use something else, then.

**Reply by 1893945** [reply]
Pyinstaller can't support？

**Reply by rokm** [reply]
> Pyinstaller can't support？

No.

**Reply by 1893945** [reply]
uhh……

Maybe I didn't express my intention clearly,

I mean that the dependent files are also in the packaging directory, but they are all placed in a specified directory, not at the same level as the exe file

**Reply by rokm** [reply]
Yes, I know. But the layout of `onedir` builds is fixed for various reasons (there's several types of search paths at play), and cannot be changed. So if you don't like it, use a different tool.
